### PR TITLE
UI: Publication year facet

### PR DESCRIPTION
**Pull request by karolina-siemieniuk-morawska**

### CODE REVIEW

File: ui/src/pages/_document.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ const MyDocument = () => (
       <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png" />
       <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png" />
       <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png" />
+      <link rel="stylesheet" href="https://unpkg.com/react-vis/dist/style.css"></link>
```
Comment (drjova) [inline ui/src/pages/_document.tsx]: 🤔 this is external link

File: ui/src/components/search/YearFacet.tsx [hunk 2]
```diff
@@ -1,77 +1,144 @@
...
 
-  const onBarMouseHover = (bar: any) => setHoveredBar({ [bar.x]: bar.y });
   const onBarMouseOut = () => setHoveredBar(null);
 
+  const resetFilters = () => {
+    setReset(!reset);
+    updateStateAndMarks(initialData);
+  };
+
+  console.log(data);
```
Comment (drjova) [inline ui/src/components/search/YearFacet.tsx]: oups

File: ui/src/pages/_document.tsx (hunk 1 above)
Comment (karolina-siemieniuk-morawska) [reply ui/src/pages/_document.tsx]: changed to import from node modules ```import 'react-vis/dist/style.css';```

File: ui/package.json [hunk 3]
```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@ant-design/cssinjs": "^1.17.2",
     "antd": "^5.10.1",
+    "lodash": "^4.17.21",
```
Comment (drjova) [inline ui/package.json]: can we just install only the lodash that we need?

Comment (karolina-siemieniuk-morawska) [reply ui/package.json]: changed to ```lodash.isequal```

**Review by drjova — APPROVED** [review]

**Pull request merged**